indexer/src/test/scala/com/expedia/www/haystack/trace/indexer/unit/ElasticSearchWriterUtilsSpec.scala

```diff
@@ -0,0 +1,32 @@
+package com.expedia.www.haystack.trace.indexer.unit
+
+import com.expedia.www.haystack.trace.indexer.writers.es.ElasticSearchWriterUtils
+import org.scalatest.{BeforeAndAfterEach, FunSpec, GivenWhenThen, Matchers}
+
+class ElasticSearchWriterUtilsSpec extends FunSpec with Matchers with GivenWhenThen with BeforeAndAfterEach {
+  var timezone = "UCT"
```

Comment:
Is this a typo ("UCT" instead of "UTC")?

Reply:
my bad good catch

Reply:
If so, it might be an indication there should be a constant :-)

Reply:
Haha, I actually didn't need it in the first place, I just couldn't figure out how to instantiate a variable with no value. This just made me google it :) haha.